Bot/Discord.Bot.Extensions/Recruiting/Commands/AdminAddTrackerCommand.cs

```diff
@@ -16,68 +16,84 @@ public class AdminAddTrackerCommand
         public static async Task Run(SocketSlashCommand command, DiscordSocketClient client, Dictionary<string, SocketSlashCommandDataOption> options, ISocketMessageChannel recruitingChannel, List<IMessage> messages, List<Team> teams)
         {
             var guildUser = (SocketGuildUser)options["username"].Value;
+            var teamName = String.Empty;
+
+            if (options.TryGetValue("team", out SocketSlashCommandDataOption teamOption))
+            {
+                teamName = teamOption.Value.ToString();
+            }
 
             // Construct new player from parameters
             var newPlayer = new Player();
             newPlayer.DiscordUser = guildUser.Nickname ?? guildUser.Username;
             newPlayer.Platform = (Platform)Enum.Parse(typeof(Platform), options["platform"].Value.ToString());
             newPlayer.PlatformId = options["id"].Value.ToString();
 
-
             if (newPlayer.Platform == Platform.Tracker && !Player.ValidateTrackerLink(newPlayer.PlatformId))
             {
-                await command.RespondAsync($"Your RL tracker link is invalid", ephemeral: true);
+                await command.FollowupAsync($"Your RL tracker link is invalid", ephemeral: true);
                 return;
             }
 
             // Is player just updating tracker link? -> Update link
-            Team updatedTeam = null;
-            foreach (var team in teams)
+            (var team, var existingPlayer) = Team.FindPlayer(teams, newPlayer.DiscordUser);
+
+            if (existingPlayer != null && !string.IsNullOrEmpty(teamName))
+            {
+                await MoveTrackedUserCommand.Run(command, client, options, recruitingChannel, messages, teams);
+                (team, existingPlayer) = Team.FindPlayer(teams, newPlayer.DiscordUser);
+            }
+            
+            if (team == null || !string.IsNullOrEmpty(teamName))
```

Comment:
need to do this for when teamName is specified and team isn't null because the move tracked user command doesn't update the tracker link

Reply:
decided to not allow admin add to move a user since we have a command for that